Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AcknowledgementViewController.swift
+//  Pods
+//
+//  Created by Michael Skiba on 11/21/16.
+//
+//
+
+import UIKit
+
+open class AcknowledgementViewController: UIViewController {
+
+    public static let defaultLicenseFormatter: (String) -> NSAttributedString
+        = { string in
+            let font = UIFont.preferredFont(forTextStyle: .body)
+            let paragraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.hyphenationFactor = 1
+            paragraphStyle.paragraphSpacing = font.pointSize / 2
+            let attributes: [String: Any] = [
+                NSFontAttributeName: font,
+                NSParagraphStyleAttributeName: paragraphStyle,
+                ]
+            return NSAttributedString(string: string, attributes: attributes)
+    }
+
+    public var viewModel = AcknowledgementViewModel(title: "", license: "") {
+        didSet {
+            updateWithViewModel()
+        }
+    }
+
+    public var licenseFormatter: (String) -> NSAttributedString = defaultLicenseFormatter
+
+    public let textView: UITextView = {
+
+        let textView = UITextView()
+        textView.isEditable = false
+        textView.alwaysBounceVertical = true
+
+        return textView
+    }()
+
+    convenience init(viewModel: AcknowledgementViewModel, licenseFormatter: @escaping (String) -> NSAttributedString, viewBackgroundColor: UIColor?) {
+        self.init()
+        self.licenseFormatter = licenseFormatter
+        self.viewModel = viewModel
+        if let backgroundColor = viewBackgroundColor {
+            textView.backgroundColor = backgroundColor
+        }
+    }
+
+    open override func loadView() {
+        view = UIView()
+        view.addSubview(textView)
+    }
+
+    open override func viewDidLoad() {
+        super.viewDidLoad()
+        updateWithViewModel()
+        configureLayout()
+    }
+
+    open override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        textView.setContentOffset(CGPoint(x: 0, y: -topLayoutGuide.length), animated: false)
```

Comment:
Interesting that `automaticallyAdjustsScrollViewInsets` didn't do this. The property is a terrible, but I'm curious why it doesn't update.